### PR TITLE
Fix API concept page issues

### PR DESCRIPTION
**Pull request by anupama-pathirage**

### CODE REVIEW

File: en/docs/apis/api-concepts.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # API Concepts
-Choreo's API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. Here are some basic concepts which will help you understand API management in Choreo as you create and manage your APIs in the [Choreo Console]({{choreo_console}}).
+Choreo API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. Here are some basic concepts which will help you understand API management in Choreo as you create and manage your APIs in the [Choreo Console]({{choreo_console}}).
```
Comment (Nashaath) [inline en/docs/apis/api-concepts.md:2]: Seems like this page also has[Choreo Console]({{choreo_console}}) leading to a broken link.

Comment (anupama-pathirage) [reply en/docs/apis/api-concepts.md]: Fixed. Thanks

**Review by Nashaath — APPROVED** [review]

File: en/docs/apis/api-concepts.md [hunk 2]
```diff
@@ -22,30 +22,29 @@ The access token is a string that the consumer passes in an HTTP header of the A
 
 ## Rate limiting
 Rate limiting limits the number of permitted requests to an API within a specified timeframe. You can use a rate-limiting strategy for the following use cases:
-    - Protect your APIs from common types of security attacks, such as certain types of Denial of Service (DoS) attacks 
-    - Regulate traffic according 
+
+- Protect your APIs from common types of security attacks, such as certain types of Denial of Service (DoS) attacks
+- Regulate traffic according to the infrastructure availability
```
Comment (anupama-pathirage) [inline en/docs/apis/api-concepts.md:27]: @shaniR Can you pls double check this? Whether this is the correct fix for missing part.

Comment (shaniR) [reply en/docs/apis/api-concepts.md:27]: Yes this is  the missing part.

**Pull request merged**